src/Assert.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\View;
+
+use Laminas\Mvc\View\Exception\AssertionFailedException;
+use Webmozart\Assert\Assert as WebmozartAssert;
+
+/** @internal  */
+final class Assert extends WebmozartAssert
```

Comment:
not sure if its a good idea to actualy extend this, but thats not a real blocker for me

Reply:
From the webmozart/assert docs:

> The Assert class comes with a few methods, which can be overridden to change the class behaviour. You can also extend it to add your own assertions.

So, I think it's fine. We could just throw an exception, but this approach gives us consistent assertion of arguments within a factory.